packages/mobile_app/lib/common/card/big_card/card_with_button.dart

```diff
@@ -46,6 +46,7 @@ class PredictionCardWithButton extends StatelessWidget {
           top: 16.0,
         ),
         scrollDirection: Axis.vertical,
+        physics: NeverScrollableScrollPhysics(),
```

Comment:
Isn't it out of scope of this change?

Reply:
It is, but it was interfering in the scrolling of ambiance